ghc-lib-gen/src/Ghclibgen.hs

```diff
@@ -296,9 +296,9 @@ cabalPackageDb ghcNumericVersion = do
           home </> ".cabal/store" </> ghcVersion </> "package.db"
   pure cabalPackageDb
 
-ghcPackagePath :: String -> String -> String
+ghcPackagePath :: String -> String -> IO String
```

Comment:
i played with it both ways. it would be convenient to leave it for the moment if you don't mind. if you feel strongly about it we can change this when my pending diff stack is cleared (there's a big refactor in the queue).

Reply:
Ah sure, nw